zwave_me_server/config.yaml

```
@@ -13,10 +13,12 @@ ports:
   8084/tcp: 8084
 options:
   device: /dev/ttyAMA0
+  configjson_device_replace: false
 schema:
   device: device(subsystem=tty)
+  configjson_device_replace: bool?
 webui: http://[HOST]:[PORT:8083]
 ports_description:
   8083/tcp: Z-Way UI
   8084/tcp: Webif UI
-image: "ghcr.io/z-wave-me/{arch}-ha-z-wave-me-addon"
+image: "ghcr.io/msazanov/{arch}-ha-z-wave-me-addon"
```

should be reverted to z-wave-me before making a PR

```suggestion
image: "ghcr.io/z-wave-me/{arch}-ha-z-wave-me-addon"
```